spec/tapioca/cli/dsl_spec.rb

```diff
@@ -1371,6 +1371,67 @@ def body=(body); end
           RBI
         end
 
+        it "must respect `skip_constant` option" do
+          @project.write!("lib/post.rb", <<~RB)
+            class Post; end
+          RB
+
+          @project.write!("lib/job.rb", <<~RB)
+            class Job; end
+          RB
+
+          @project.write!("lib/compilers/foo/compiler.rb", <<~RB)
+            require "job"
+            require "post"
+
+            module Foo
+              class Compiler < Tapioca::Dsl::Compiler
+                extend T::Sig
+
+                ConstantType = type_member { { fixed: Module } }
+
+                sig { override.void }
+                def decorate
+                  root.create_path(constant) do |job|
+                    job.create_module("FooModule")
+                  end
+                end
+
+                sig { override.returns(T::Enumerable[Module]) }
+                def self.gather_constants
+                  [Post, Job]
+                end
+              end
+            end
+          RB
+
+          result = @project.tapioca("dsl --skip-constant Job")
```

Comment:
Let's add a test with more than one value for `--skip-constant` so we do not accidentally break this in the future.